### PR TITLE
syspolicy: add Allow LAN Access visibility key

### PR DESCRIPTION
**Pull request by clairew**
Fixes tailscale/corp#15594

### CODE REVIEW

File: util/syspolicy/policy_keys.go [hunk 1]
```diff
@@ -24,6 +24,7 @@ const (
 	UpdateMenuVisibility      Key = "UpdateMenu"
 	RunExitNodeVisibility     Key = "RunExitNode"
 	PreferencesMenuVisibility Key = "PreferencesMenu"
+	AllowLANAccessVisibility  Key = "AllowLANAccess"
```
Comment (sailorfrag) [inline util/syspolicy/policy_keys.go:27]: I think this should be a always/never/user-decides, not a show/hide?

Comment (clairew) [reply util/syspolicy/policy_keys.go:27]: good point. @agottardo thoughts? thanks!

Comment (agottardo) [reply util/syspolicy/policy_keys.go:27]: @clairew agreed! This should be always/never/u-d.

**Comment by agottardo** [comment]
I would also call the key "ExitNodeAllowLANAccess" for clarity, maybe?

**Review by agottardo — APPROVED** [review]

**Pull request merged**